src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -16,18 +16,257 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
-    internal sealed class SourceOrdinaryMethodSymbol : SourceOrdinaryMethodSymbolBase
+    internal abstract class SourceOrdinaryMethodSymbol : SourceOrdinaryMethodSymbolBase
     {
-        private readonly TypeSymbol _explicitInterfaceType;
+        private sealed class SourceOrdinaryMethodSymbolCommon : SourceOrdinaryMethodSymbol
```

Comment:
>Common [](http://example.com/codeflow?start=55&length=6)

I think we used terms "Simple" and "Complex" for parameters. Consider using them here as well

Reply:
Done.